tests/python/relay/test_op_level2.py

```diff
@@ -58,5 +57,134 @@ def test_conv2d_infer_type():
 
 
 
+def test_resize_infer_type():
+    ib = relay.ir_builder.IRBuilder()
+    n, c, h, w = tvm.var("n"), tvm.var("c"), tvm.var("h"), tvm.var("w")
+    x = ib.param("x", relay.ty.TensorType((n, c, h, w), "int8"))
+    th, tw = tvm.var("th"), tvm.var("tw")
+
+    with ib.function(x) as func:
```

Comment:
create test_op_levelk(where k equals the corresponding level, and put the files onto their respective positions.